### ISSUE TITLE
Missing required lables in registration and login forms

### ISSUE THREAD

**Issue by mauromsl** [BirkbeckCTP/clean#9]
Fields that are required on the login and registration form don't indicate so with a label.
Additionally they should also set the `aria-required` attribute to `true`

WCAG 3.3.2
